### ISSUE TITLE
Allow for captions

### ISSUE THREAD

**Issue by simsong** [saimn/sigal#379]
It would be great to have captions for the images appear on the gallery. Captions could be provided by either a .txt file with the same basename as the .jpg, or else embedded in the EXIF.

**Comment by saimn** [comment]
That's already possible (http://sigal.saimon.org/en/latest/image_information.html), though maybe not supported by default by all themes. For the use of EXIF metadata in particular you may have to customize the theme's template files.

**Comment by simsong** [comment]
I'm trying to get it to work. It is hard to find because a search for "caption" on the signal website turns up nothing.  The [web page you pointed me at](http://sigal.saimon.org/en/latest/image_information.html) does not make it obvious how to enable captions. I'm working on it...

**Comment by simsong** [comment]
When you say customize the theme's template files, do you mean in the source code? Is there a way to simply overrride the themes?

**Comment by saimn** [comment]
Yes the documentation about the themes could be improved... A lot about the available attributes is described directly by the API docs (http://sigal.saimon.org/en/latest/themes.html#documentation-of-sigal-s-main-classes). And to override the themes you need to copy the theme directory locally. You can specify the path to your local theme in the settings file or on the command line.

**Comment by shacker** [comment]
In my case, I'm trying to alter the display of the caption so that it shows the date but not the time. Unfortunately it looks like there's some javascript involved - the only thing present in html is `<div class="pswp__caption__center"></div>` . I'm digging around trying to figure out how to alter that, but don't see any documentation on that.

**Comment by saimn** [comment]
In that case it's managed by photoswipe, which can be customized in various ways, e.g. https://photoswipe.com/caption/
